.github/workflows/package.yml

```diff
@@ -1,31 +0,0 @@
-name: Package Tarballs
-on:
-  # Run this workflow on branch main pushes and tag pushes that match v* (eg. `v1.2.3`)
-  push:
-    branch:
-      - 'main'
-    tags:
-      - 'v*'
-
-jobs:
-  package:
-    name: Package Tarballs
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Check out code
-      uses: actions/checkout@v2
-
-    - name: Install Helm
-      uses: azure/setup-helm@v1
-      with:
-        version: v2.16.6
-
-    - name: Package tarballs
-      run: mkdir -p package; helm package --save=false --destination ./package ./conjur-oss
-
-    - name: Add package to artifacts
-      uses: actions/upload-artifact@v1
-      with:
-        path: ./package
-        name: package
```

Comment:
This is a github action that wouldn't be pulled across (`.github` repo excludes actions).  I believe the `.github/workflows` directory should be preserved.